docs/architecture/data_loading.rst

```diff
@@ -271,17 +270,17 @@ On a plus side this maps nicely to the way things work in
 allows for greatest variety of storage regimes
 
 -  bands/time slices split across multiple files
--  bands stored in one files, one file per time slice
+-  bands stored in one fil, one file per time slice
 -  stacked files that store multiple time slices and all the bands
 
 On the other hand this way of partitioning code leads to less than
 optimal I/O access patterns. This is particularly noticeable when using
 “stacked files” (a common use case on the NCI installation of the
-datacube) while doing “pixel drill” type of access.
+dacube) while doing “pixel drill” type of access.
```

Comment:
definitely "datacube"

Reply:
yes, didn't notice that one, sorry, have updated